### ISSUE TITLE
Nesting NotFoundError thrown is erroneously caught in meta file lookup

### ISSUE THREAD

**Issue by DorianKwan** [launchcodedev/app-config#160]
# Problem
No explicit error message when type declarations fail to be generated due to a meta file extension failure.

Existing message
`[app-config][WARN] No files generated - did you add the correct meta properties?`

# Expected Behaviour
When type generations fail due to a failed file extension, write an error message indicating this to STDOUT.

# Steps to Reproduce
Extend an app-config meta file within a new meta file and add a typo in the targeted file extension. 

Run the generate command and notice the error message and that the type declarations haven't been generated.

**Comment by joelgallant** [comment]
@DorianKwan what file extension did you observe this with? Quicktype should throw errors for unsupported types.

**Comment by DorianKwan** [comment]
Sorry, I should've been more clear.  In this case I'm referring to extending a root meta file

**Comment by joelgallant** [comment]
I guess I'm unclear still. Doing what you suggested results in `QuickTypeError: Unknown output language st` for me.

**Comment by gregnr** [comment]
@DorianKwan Best way to help reproduce is always an example. @joelgallant this is what happened today:

_./services/my-service/.app-config.meta.yml:_
```yaml
$extends: ../../app-config.meta.yml # notice there was a typo here - we forgot the leading dot '.app-config.meta.yml'

generate:
   ...

...
```
When attempting to generate typings using `yarn app-config generate`, we get the error:
```
[app-config][WARN] No files generated - did you add the correct meta properties?
```
After discovering the true issue (typo on extended file), this error makes sense because it couldn't parse the meta file correctly.
But ideally because the path '../../app-config.meta.yml' didn't exist, we would expect an error from the `$extends` extension indicating that it couldn't extend because the file didn't exist.

**Comment by joelgallant** [comment]
Got it. Yeah, I vaguely knew this was possible, because meta file loading will ignore a `NotFound` error (for good reason, meta file isn't required). But if that error was bubbled up while parsing, it shouldn't be ignored.

**Comment by joelgallant** [comment]
Released in `2.6.1`
